### PR TITLE
PIA-1598: Add signup credentials screen

### PR DESCRIPTION
**Pull request by kp-said-rehouni**
**Summary**
This PR implements the credentials screen for Sign up on tvOS

### CODE REVIEW

File: PIA VPN-tvOS/SignupCredentials/CompositionRoot/SignupCredentialsFactory.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    static var userAccount: UserAccount?
+    
+    static func makeSignupCredentialsView() -> SignupCredentialsView {
+        guard let credentials = userAccount?.credentials else {
+            fatalError("Can't be created without user credentials")
+        }
+        
+        return SignupCredentialsView(credentials: credentials) {
+            NotificationCenter.default.post(name: .PIAAccountDidLogin, object: nil)
+        }
```
Comment (kp-laura-sempere) [inline PIA VPN-tvOS/SignupCredentials/CompositionRoot/SignupCredentialsFactory.swift:21]: Not a big deal, just perhaps for readability or understanding better each of the dependencies here, we could name the `action` parameter that is added to this View?

**Review by kp-laura-sempere — DISMISSED** [review]
It looks very good 👍 

I just added one comment, but I don't see it as a blocker for merging

File: PIA VPN-tvOS/SignupCredentials/CompositionRoot/SignupCredentialsFactory.swift (hunk 1 above)
Comment (kp-said-rehouni) [reply PIA VPN-tvOS/SignupCredentials/CompositionRoot/SignupCredentialsFactory.swift:21]: Good point! Let me update it.
Thank you @kp-laura-sempere!

**Review by kp-laura-sempere — APPROVED** [review]
Thank you @kp-said-rehouni , it looks great 🚀

**Pull request merged**